labelSchema.js

```diff
@@ -83,6 +83,68 @@ function getFRACountryValue() {
   };
 }
 
+
+// this function generates the last field of the labels for FRA records
+// 1.  use the region name if the record is a in the French overseas, eg - Saint-Denis, Reunion
+// 2.  use dependency name if not null
+// 3.  use country name, eg - Paris, France
+function getFRACountryValue() {
+  const _overseas = getFirstProperty(['region', 'dependency', 'country']);
+  const _default = getFirstProperty(['dependency', 'country']);
+  return (record) => {
+    if (!_.isEmpty(record.region_a) && _.includes(FRA_OVERSEAS, record.region_a[0])) {
+      return _overseas(record);
+    }
+    return _default(record);
+  };
+}
```

Comment:
Copy/Paste to delete (L71-L84)

Reply:
:+1: this section seems to have been duplicated